docs/source/index.rst

```diff
@@ -22,16 +22,14 @@ AssessPy package
    Source Code <https://github.com/ccao-data/assesspy>
 
 AssessPy is a software package for Python developed by the Cook County
-Assessor's (CCAO) Data Department. The codebase for the CCAO's CAMA system
-uses a wide range of functions regularly, and packaging these functions
-streamlines and standardizes their use. The CCAO is publishing this package
-to make it available to assessors, reporters, and citizens everywhere.
-
-For assessors, we believe that this package will reduce the complexity
-of calculating ratio statistics and detecting sales chasing. We also
-believe that reporters, taxpayers, and members of academia will find
-this package helpful in monitoring the performance of local assessors
-and conducting research.
+Assessor’s (CCAO) Data Department. It contains many of the functions necessary
+to perform a standard
+`sales ratio study <https://www.iaao.org/wp-content/uploads/Standard_on_Ratio_Studies.pdf>`_
+
+For assessors, we believe that this package will reduce the complexity of
+calculating ratio statistics and detecting sales chasing. We also hope that
+reporters, taxpayers, and members of academia will find this package helpful
+in monitoring the performance of local assessors and conducting research.
```

Comment:
This text was out-of-date compared to the README, so just quickly copied it over.